### ISSUE TITLE
open as root stopped working

### ISSUE THREAD

**Issue by rjv23** [linuxmint/nemo#3234]
```
 * Nemo version (nemo --version) - 5.6.4
 * Is issue with desktop or windowed nemo? desktop
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) fedora 38
 * Graphics hardware *and* driver used - nvidia and intel - nouveau
 * 64 bit 
 ```

**Issue**
open as root will not open new elevated privileges window

**Steps to reproduce**

right click on open space, open as root, password identification comes up, password entered but no new window opens and old window without privileges

**Expected behaviour**

**Other information**

**Comment by leigh123linux** [comment]
I works fine here.

```
$ inxi -SG
System:
  Host: mpd-pc Kernel: 6.2.11-300.fc38.x86_64 arch: x86_64 bits: 64
    Desktop: Cinnamon v: 5.6.8 Distro: Fedora release 38 (Thirty Eight)
Graphics:
  Device-1: Intel GeminiLake [UHD Graphics 600] driver: i915 v: kernel
  Display: x11 server: X.Org v: 1.20.14 with: Xwayland v: 22.1.9 driver: X:
    loaded: modesetting unloaded: fbdev,vesa dri: iris gpu: i915
    resolution: 3840x2160~60Hz
  API: OpenGL v: 4.6 Mesa 23.0.3 renderer: Mesa Intel UHD Graphics 600 (GLK
    2)
```

**Comment by Zerophase** [comment]
It's an issue with updating versions of Cinnamon.

**Comment by Zerophase** [comment]
If I run Nemo from a command prompt the only log output received is:
`Could not parse arguments: Cannot open display:`

**Comment by leigh123linux** [comment]
Are you trying to use nemo in a wayland session.

**Comment by Zerophase** [comment]
Nope, just X11. I do have Wayland installed as a dependency for other packages.

**Comment by Zerophase** [comment]
I think it's an issue with the difference between how Mint, and other distros update Cinnamon.

**Comment by mtwebster** [comment]
> If I run Nemo from a command prompt the only log output received is:
Could not parse arguments: Cannot open display:

Is this while trying to run as root, or just running 'nemo'?

Does your environment contain a DISPLAY variable? (typically `DISPLAY=:0`)

Do other graphical programs run ok, as root or otherwise?

**Comment by Zerophase** [comment]
> > If I run Nemo from a command prompt the only log output received is:
> > Could not parse arguments: Cannot open display:
> 
> Is this while trying to run as root, or just running 'nemo'?
Just running Nemo

> Does your environment contain a DISPLAY variable? (typically `DISPLAY=:0`)
Yeah, it sets `DISPLAY=:0`

> Do other graphical programs run ok, as root or otherwise?

Others run fine.

The only possible user action that might have done this was restoring from a backup awhile back. Pretty sure, I used rsync, or restored from a borg archive. Pretty sure, I restored properly, though. Are there any permissions for files I should double check?

**Comment by leigh123linux** [comment]
Does this run?

```
pkexec lightdm-settings
```

**Comment by Zerophase** [comment]
Just installed, and that does not run. `sudo lightdm-settings` does.

**Comment by claudiux** [comment]
I just upgraded from Fedora 37 to Fedora 38 (Cinnamon 5.8.2) and opening as root any folder, including Desktop, works fine.

**Comment by Zerophase** [comment]
I had upgrades that did not break stuff either. I think it's more of a function of having a system running for a long time.

**Comment by nazar-pc** [comment]
Upgraded to Ubuntu 24.04 and installed Nemo on custom X11 session and also getting `Could not parse arguments: Cannot open display:`. No idea if it worked before, Ubuntu 24.04 shipped Nemo 6.0.2 and I don't have full Cinnamon stack installed here.

**Comment by v-knox** [comment]
has this problem not been solved yet? 
>nxi -SG
System:
  Host: vnoxlaptop Kernel: 6.14.0-37-generic arch: x86_64 bits: 64
  Desktop: Cinnamon v: 6.4.8 Distro: Linux Mint 22.2 Zara
Graphics:
  Device-1: AMD Lucienne driver: amdgpu v: kernel
  Device-2: IMC Networks HD Camera driver: uvcvideo type: USB
  Display: x11 server: X.Org v: 21.1.11 with: Xwayland v: 23.2.6 driver: X:
    loaded: amdgpu unloaded: fbdev,modesetting,vesa dri: radeonsi gpu: amdgpu
    resolution: 1920x1080~60Hz
  API: EGL v: 1.5 drivers: kms_swrast,radeonsi,swrast
    platforms: gbm,x11,surfaceless,device
  API: OpenGL v: 4.6 compat-v: 4.5 vendor: amd mesa
    v: 25.0.7-0ubuntu0.24.04.2 renderer: AMD Radeon Graphics (radeonsi renoir
    ACO DRM 3.61 6.14.0-37-generic)<


im also experiencing this issue on nemo 6.4.5 and i also didnt get a return on `pkexec lightdm-settings` but sudo lightdm-settings worked properly
